l5kit/l5kit/tests/dataset/dataset_test.py

```diff
@@ -31,6 +42,24 @@ def check_torch_loading(dataset: Dataset) -> None:
 def test_dataset_rasterizer(
     rast_name: str, dataset_cls: Callable, zarr_dataset: ChunkedDataset, dmg: LocalDataManager, cfg: dict
 ) -> None:
+    cfg = deepcopy(cfg)
+    cfg["raster_params"]["map_type"] = rast_name
```

Comment:
@lucabergamini  a bug that `rast_name` was not used

Reply:
good catch!

Reply:
I'd say you don't need the deepcopy as the fixture's scope is set to `function`